src/components/list/List.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const List = ({title, children, classname}  ) => {
```

Comment:
Omdat de list alleen zwart of wit kan zijn, kun je in plaats van de letterlijke css-class (die iemand dan moet onthouden) ervoor kiezen om een `theme` of `varient` prop mee te geven! Dus bijvoorbeeld varient `dark` of `light`, waarop het List component dan zelf bepaald welke CSS class er op het element gezet moet worden!